app/controllers/api/v1/jwt_api/people_controller.rb

```diff
@@ -0,0 +1,76 @@
+# frozen_string_literal: true
+
+module Api
+  module V1
+    module JwtApi
+      class PeopleController < JwtApiController
+        before_action :set_email
+        before_action :check_email
+        before_action :set_parent
+        before_action :set_team_name
+        before_action :set_first_name
+        before_action :set_role_title
+
+        def create
+          @person = CreateChildPerson.run!(
+            parent: @parent,
```

Comment:
nice, can we also use this for mentors / students? I like the notion of parent child, as its way more generic (even for this specific ikea (:-)) case

Reply:
Yes, we can. It just depends on the team names and roles you specify.